src/libs/actions/IOU.js

```diff
@@ -886,9 +894,12 @@ function createSplitsAndOnyxData(participants, currentUserLogin, currentUserAcco
 
     const hasMultipleParticipants = participants.length > 1;
     _.each(participants, (participant) => {
-        // In case the participant is a worskapce, email & accountID should remain undefined and won't be used in the rest of this code
-        const email = isOwnPolicyExpenseChat ? '' : OptionsListUtils.addSMSDomainIfPhoneNumber(participant.login).toLowerCase();
-        const accountID = isOwnPolicyExpenseChat ? 0 : Number(participant.accountID);
+        // In a case when a participant is a workspace, even when a current user is not an owner of the workspace
+        const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(participant);
+
+        // In case the participant is a workspace, email & accountID should remain undefined and won't be used in the rest of this code
+        const email = isOwnPolicyExpenseChat || isPolicyExpenseChat ? '' : OptionsListUtils.addSMSDomainIfPhoneNumber(participant.login).toLowerCase();
```

Comment:
Let me clarify: for these 2 conditions (`email` and `accountID`) the app checks only if a `participant` is own policy expense chat. However, the `participant` could be a stranger workspace.
Yes, it causes a bug. Because, at right side of the conditions it gets `login` and `accountID` of the participant, but the participant does not have these values, when it is a workspace. 
Actually, the app crashes when it tries to run `toLowerCase()` of `login` that is `undefined`.

Reply:
The other cases where `isOwnPolicyExpenseChat` is being used, we don't need to add `isPolicyExpenseChat`. I was investigating it and haven't found a sense.